extensions/azuremonitor/src/features/serializationFeature.ts

```diff
@@ -0,0 +1,68 @@
+import { SqlOpsDataClient, SqlOpsFeature } from 'dataprotocol-client';
+import { ClientCapabilities, RPCMessageType, ServerCapabilities } from 'vscode-languageclient';
+import { Disposable } from 'vscode';
+import * as contracts from './contracts';
+import * as azdata from 'azdata';
+import * as Utils from '../utils';
+import * as UUID from 'vscode-languageclient/lib/utils/uuid';
+
+export class SerializationFeature extends SqlOpsFeature<undefined> {
```

Comment:
This doesn't seem to be doing anything, why do you have this here?

Reply:
This is required to export to CSV, JSON, etc unfortunately.